topi/python/topi/x86/conv2d.py

```diff
@@ -27,7 +27,7 @@
 from .. import generic, tag
 from .. import nn
 from ..util import get_const_tuple
-from ..nn.conv2d import conv2d, conv2d_NCHWc, \
+from ..nn.conv2d import conv2d, conv2d_NCHWc, _group_conv2d_nchw, \
```

Comment:
We'd better not importing an internal function. I think in nn.py we don't need an extra internal function for group_conv2d_nchw.

Reply:
Thanks! I've removed this update since there is no need to register TOPI compute for group_covn2d_nchw now